### PR TITLE
Update Kafka Hub Readme

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] ~Linked to an issue~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~

### CODE REVIEW

File: examples/kafka-hub/README.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ As mentioned above implementation is based on Kafka message broker. Therefore, t
 
 As you can see the current implementation is relying on two different technologies. An IdP is used to handle any authentication and authorization request. This would be the normal case in any production environment. For the other quality of services such as message persistence, subscription management, etc the implementation depends on Kafka message broker. 
 
+`Event Consolidator` is a supportive service which handles the race-conditions related to `topic` and `subscription` management.
+
+Following diagram depict how the kafka topics are utilized by the `Hub` and the `Event Consolidator`.
```
Comment (chamil321) [inline examples/kafka-hub/README.md:34]: depicts*

Comment (ayeshLK) [reply examples/kafka-hub/README.md]: Fixed with [c21341e2dbc37b274b42c4446c029878327f6212](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/557/commits/c21341e2dbc37b274b42c4446c029878327f6212)

File: examples/kafka-hub/README.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ As mentioned above implementation is based on Kafka message broker. Therefore, t
 
 As you can see the current implementation is relying on two different technologies. An IdP is used to handle any authentication and authorization request. This would be the normal case in any production environment. For the other quality of services such as message persistence, subscription management, etc the implementation depends on Kafka message broker. 
 
+`Event Consolidator` is a supportive service which handles the race-conditions related to `topic` and `subscription` management.
+
```
Comment (shafreenAnfar) [inline examples/kafka-hub/README.md:33]: I think we should explain how we can get race conditions as well.

Comment (ayeshLK) [reply examples/kafka-hub/README.md:33]: Fixed with [8d94ffb0661a35177eb6e84ff436f147a3ff32fe](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/557/commits/8d94ffb0661a35177eb6e84ff436f147a3ff32fe)

**Review by chamil321 — APPROVED** [review]

**Pull request merged**